dataland-inbound-proxy/config/utils/keycloak.conf

```diff
@@ -0,0 +1,19 @@
+location /keycloak/js {
+  proxy_pass http://keycloak:8080/keycloak/js;
+  include utils/proxy.conf;
+}
+
+location /keycloak/realms {
+  proxy_pass http://keycloak:8080/keycloak/realms;
+  include utils/proxy.conf;
+}
+
+location /keycloak/resources {
+  proxy_pass http://keycloak:8080/keycloak/resources;
+  include utils/proxy.conf;
+}
+
```

Comment:
Wir sollten einen Test schreiben, der überprüft, dass weder /keycloak/admin noch /keycloak/realms/master funktionieren.

Reply:
keycloak/admin ist jetzt da - keycloak/realms/master fehlt noch.

Reply:
keycloak/realms/master ist glaube ich vorhanden in dem testfile:
it(`Master Realm not reachable from remote`, () => {
    checkThatUrlResolvesToErrorPage("/keycloak/realms/master");
  });

Reply:
ok